templates/angular/igx-ts/custom-templates/tree-grid-finjs/files/src/app/themes/_mixins.scss

```diff
@@ -0,0 +1,19 @@
+@mixin scrollbar-love($scrollbar-color: null) {
```

Comment:
This should be moved under `__path__` folder - currently, if you try to add 2 or more `FinTech` components, they'll conflict, because files under `app/` will already exist